src/Raven.Studio/wwwroot/Content/css/pages/pull-replication.less

```diff
@@ -1,5 +1,13 @@
 ﻿.edit-pull-replication-hub-task {
     
+    #replicationAccessList {
```

Comment:
.replication-access-list and please use class not id.

Reply:
done - renamed and used class instead of 'id'.
(but - this can be 'id' - it only exists in one single place on the page)